src/style.css

```diff
@@ -0,0 +1,385 @@
+* {
```

Comment:
This file is huge. 

The idealogy  of React follows that your application consists of individual blocks named `components`

Then you also probably need to separate your styles. Consider using css modules: https://vitejs.dev/guide/features.html#css-modules